dotnet/StachExtensions/FactSet.Protobuf.Stach.Extensions/V2/RowOrganizedStachExtension.cs

```diff
@@ -136,8 +138,26 @@ private Models.Table GenerateTable(RowOrganizedPackage.Types.Table table)
 
             foreach (var metadataItem in table.Data.TableMetadata)
             {
-                var metadataValue = StachUtilities.ValueToString(metadataItem.Value.Value);
-                finalTable.Metadata.Add(metadataItem.Key, metadataValue);
+                var metadataValueString = StachUtilities.ValueToString(metadataItem.Value.Value);
+                var metadataValue = metadataItem.Value.Value;
+
+                finalTable.Metadata.Add(metadataItem.Key, metadataValueString);
+
+                List<Google.Protobuf.WellKnownTypes.Value> valuesList = new List<Google.Protobuf.WellKnownTypes.Value>();
+                if (metadataValue.KindCase.ToString() == "ListValue")
+                {
+                    foreach (Google.Protobuf.WellKnownTypes.Value val in metadataValue.ListValue.Values)
+                    {
+                        valuesList.Add(val);
+                    }
+                }
+                else if (metadataValue.KindCase.ToString() == "StringValue")
+                {
+                    string val = metadataValue.ToString().Trim('\"');
```

Comment:
Why do we need to Trim here?

Reply:
Im not clear on this else  condition. If the  metadataValue.KindCase is StringValue you can directly add it to the valuesList    right?

Reply:
Trimming was needed because the conversion `toString()` was changing the structure of `val` somehow. Converting to string wasn't actually necessary though so I removed all of that logic here https://github.com/factset/stach-extensions/pull/18/commits/e82b348b51d136e8e5ce02f3b53ae6d52b49e930